### PR TITLE
PWN - 3868 add ci build artifacts

### PR DESCRIPTION
**Pull request by davran1001**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-3868

## Description of Work

[ what work was done in this ticket ]

## Screenshots (Optional)

Screenshot #1 | Screenshot #2 | Screenshot #3
:------------:|:-------------:|:------------:
[Paste Pic 1] | [Paste Pic 2] | [Paste Pic 3]

### CODE REVIEW

**Review by kabramovp2p — APPROVED** [review]

File: .github/workflows/android-release-workflow.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+jobs:
+  android-lint:
+    name: "[Github Actions] - Run android ktlint check"
```
Comment (1sakov) [inline .github/workflows/android-release-workflow.yml:10]: let's remove `android` in all jobs' description

File: .github/workflows/android-release-workflow.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+          FIREBASE_APP_ID: ${{ secrets.FIREBASE_APP_ID }}
+        run: ./gradlew assembleRelease appDistributionUploadRelease
+
+      - name: Upload artifact to Slack
+        uses: ./.github/actions/setup-slack
+        with:
+          bot-token: ${{ secrets.SLACK_BOT_TOKEN }}
+          channel-id: ${{ secrets.SLACK_CHANNEL_ID }}
+          build-type: "release"
+          artifact-type: "apk"
```
Comment (1sakov) [inline .github/workflows/android-release-workflow.yml:103]: we need .aab not apk

**Review by 1sakov — CHANGES_REQUESTED** [review]

File: .github/workflows/android-release-workflow.yml [hunk 3]
```diff
@@ -0,0 +1,103 @@
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
```
Comment (gslevinkov) [inline .github/workflows/android-release-workflow.yml:7]: release workflow should work on release branches

File: .github/workflows/android-release-workflow.yml [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+      - name: Create and fill cert.pem certificate for SSL pinning
+        env:
+          SSL_CERTIFICATE: ${{ secrets.SSL_CERTIFICATE }}
+        run: echo $SSL_CERTIFICATE -> app/src/main/res/raw/cert.pem
+
+      - name: Run unit tests
+        run: ./gradlew testDebug
+
+  android-build-artifacts:
+    name: "[Github Actions] - Run android build feature artifacts"
```
Comment (gslevinkov) [inline .github/workflows/android-release-workflow.yml:49]:
```suggestion
    name: "[Github Actions] - Run android build release artifacts"
```

File: .github/workflows/android-release-workflow.yml [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+        with:
+          auth-token: ${{ secrets.SENTRY_AUTH_TOKEN }}
+          auth-dsn: ${{ secrets.SENTRY_AUTH_DSN }}
+
+      - name: Create keystore file
+        uses: ./.github/actions/setup-keystore
+        with:
+          keystore-secret: ${{ secrets.KEYSTORE_SECRET }}
+          keystore-passphrase: ${{ secrets.KEYSTORE_PASSPHRASE }}
+      - name: Upload artifact to Firebase
```
Comment (gslevinkov) [inline .github/workflows/android-release-workflow.yml:91]:
```suggestion
          keystore-passphrase: ${{ secrets.KEYSTORE_PASSPHRASE }}
          
      - name: Upload artifact to Firebase
```

File: .github/workflows/android-release-workflow.yml (hunk 2 above)
Comment (gslevinkov) [reply .github/workflows/android-release-workflow.yml:103]: moreover, we need apk + aab in slack and firebase

**Pull request merged**